spec/tapioca/cli/dsl_spec.rb

```diff
@@ -364,6 +374,24 @@ module GeneratedFoo; end
         refute_path_exists("#{outdir}/user.rbi")
       end
 
+      it "removes stale RBIs properly when running in parallel" do
+        FileUtils.mkdir_p("#{outdir}/baz")
+        FileUtils.mkdir_p("#{outdir}/namespace")
+
+        FileUtils.touch("#{outdir}/baz/role.rbi")
+        FileUtils.touch("#{outdir}/job.rbi")
+        FileUtils.touch("#{outdir}/post.rbi")
+        FileUtils.touch("#{outdir}/namespace/comment.rbi")
+
```

Comment:
Can we still add a couple of stale RBI files here in any case? I know it does not affect the case we are currently testing for, but it might catch against something we might end up introducing at some point in the future.